configure.ac

```diff
@@ -140,8 +140,8 @@ AC_ARG_ENABLE(tests,
 
 AC_ARG_ENABLE(gui-tests,
     AS_HELP_STRING([--disable-gui-tests],[do not compile GUI tests (default is to compile if GUI and tests enabled)]),
-    [use_gui_tests=$enableval],
-    [use_gui_tests=$use_tests])
+    [use_gui_tests=no],
+    [use_gui_tests=no])
```

Comment:
Why did you change it from a variable?

Reply:
This must have been removed sorry. I tried to disable GUI builds in configure script initially, then realised that CI had  the option.